src/int.cr

```diff
@@ -486,6 +486,51 @@ struct Int
     io << type
   end
 
+  # Prints this integer as a binary value in a human-readable format using
+  # multiples of 1024 as prefixes.
+  #
+  # Values with binary measurements such as computer storage (e.g. RAM size) are
+  # typically expressed using unit prefixes based on 1024 (instead of multiples
+  # of 1000 as per SI standard). This method by default uses an extended range
+  # of the JEDEC units (`K`, `M`, `G`, `T`, `P`, `E`, `Z`, `Y`) which equals to
+  # the prefixes of the SI system except for uppercase `K` and is based on
+  # powers of 1024. The IEC standard prefixes (`ki`, `Mi`, `Gi`, `Ti`, `Pi`,
+  # `Ei`, `Zi`, `Yi`) will be used if *standard* is `:IEC`.
+  #
+  # ```
+  # 1.humanize_bytes                          # => "1B"
+  # 1024.humanize_bytes                       # => "1.0KB"
+  # 1536.humanize_bytes                       # => "1.5KB"
+  # 524288.humanize_bytes                     # => "512KB"
+  # 1073741824.humanize_bytes(standard: :IEC) # => "1.0GiB"
+  # ```
+  #
+  # See `Number#humanize` for more details on the behaviour and arguments.
+  def humanize_bytes(io : IO, precision : Int = 3, separator = '.', *, significant : Bool = true, standard : Symbol = :JDEC) : Nil
```

Comment:
`:JDEC` -> `:JEDEC`

Reply:
Thanks 😉

Reply:
If only we had typsafe enums in the language...

Reply:
I considered that, but honestly I don't think it's worth adding an enum or this. It just distinguishes `:IEC` and anything else. A simple bool flag would do. I'm just using a symbol to be open for potential extensions (and can easily be changed to an enum).

Reply:
Yes, please.